docker/generic/start_proxy.py

```diff
@@ -932,6 +932,19 @@ def make_argparser():
         in grpc-json transcoding. This is to support HTML 2.0<https://tools.ietf.org/html/rfc1866#section-8.2.1>.
         Set this flag to true to disable this feature.
         ''')
+
+    parser.add_argument(
+        '--transcoding_match_unregistered_custom_verb', action='store_true',
+        help='''
+        According to the http template[1], the custom verb is **":" LITERAL** at the end of http template.
+
+        For a request with */foo/bar:baz* and *:baz* is not registered in any url_template, here is the behavior change
+         - if the field is not set, *:baz* will not be treated as custom verb, so it will match **/foo/{x=*}**.
```

Comment:
Edit bullet points to make behavior clearer

- "By default, ..."
- "If the **flag** is set to true, ..."

Reply:
Done